todo/flutter/.flutter-plugins

```diff
@@ -1 +1 @@
-path_provider=/Users/joeconway/.pub-cache/hosted/pub.dartlang.org/path_provider-0.2.1+1/
+path_provider=/Users/jon/Code/flutter/.pub-cache/hosted/pub.dartlang.org/path_provider-0.4.1/
```

Comment:
I imagine this file can get moved to gitignore? I am not familiar enough maybe @anachlas knows?